crates/router/src/core/payments/flows/session_flow.rs

```diff
@@ -955,7 +955,7 @@ fn create_paypal_sdk_session_token(
                     connector: connector.connector_name.to_string(),
                     session_token: paypal_sdk_data.data.client_id,
                     sdk_next_action: payment_types::SdkNextAction {
-                        next_action: payment_types::NextActionCall::Confirm,
+                        next_action: payment_types::NextActionCall::PostSessionTokens,
```

Comment:
Yes, this will go in sandbox after SDK sandbox deployment.

Reply:
> Yes, this will go in sandbox after SDK sandbox deployment.

What's our strategy for production rollout of this change, would it be rolled out after SDK deployment on production as well?